docs/en/week12/12-2.md

```diff
@@ -0,0 +1,184 @@
+---
+lang-ref: ch.12-2
+title: Low Resource Machine Translation
+lecturer: Marc’Aurelio Ranzato
+authors: Angela Teng, Joanna Jin
+date: 17 May 2021
+---
+
+# 12.2
+## Wed 21 Apr – (Invited) Lecture 12: Guest Lecture by Marc'Aurelio Ranzato
+
+This week's lecture was a guest lecture by [Marc'Aurelio Ranzato](https://ai.facebook.com/people/marc-aurelio-ranzato/), who is a research scientist and manager at the Facebook AI Research (FAIR) lab, where he works to enable machines to learn with weaker supervision and to efficiently transfer knowledge across tasks. The first part of this document focuses on understanding low resource machine translation, and the second half discusses potential domain mismatches in machine learning and machine translation.
+
+## ML Perspective on Low Resource MT
+We start with standard machine learning algorithms can be applied in the realm of machine translation translation. Then, we take a deeper look into understanding different perspectives on this application. Our setting consists of multiple languages and multiple domains, but eventually we want to maximize translation accuracy of a certain domain in a language class.
+
+<center>
+<img src="{{site.baseurl}}/images/week12/12-2/figure1.png" style="background-color:#DCDCDC;" /><br>
+</center>
+
+Taking a deeper look at the type of data we are presented with in machine translation, we can better understand how these applications can be mapped to machine learning techniques. For example, if we have a parallel dataset in the space of machine translation, then we have its equivalen in supervised learning. Second, we could have monolingual data in machine translation--this translates to semi-supervised learning within the machine elarning framework. Third, if we have multiple language paris in machine translation, we can closely compare this to multi-task learning within the ML space. And lastly, if we have multiple domains in machine translation, we can compare this to domain adaptation in machine learning. When you have many domains, you naturally also want to use different domain adaptation techniques
+
+
+## Case Studies
+**Translating from English to Nepali:** Let's start with a simple case study within the realm of supervsied learning. For example, let's say that we have a sentence in English and we want to trasnslate it to Nepali. This is similar to multitask learning in the sense that you have one task, and then you add another task you're interested in. Since we have multiple domains we can start thinking about domain adaptation techniques and analyzing which domain adaptation techniques are applicable to machine translation.
+
+<center>
+<img src="{{site.baseurl}}/images/week12/12-2/figure2.png" style="background-color:#DCDCDC;" /><br>
+</center>
+
+Beginning with a supervised learning setting, we can use the aforementioned example of translation. Our dataset is
+$$D = \{(x,y)_i\}_{i=1, ..., N}$$
```

Comment:
You need an empty line before and after an equation.
Check out [the instructions](https://atcold.github.io/pytorch-Deep-Learning/en/about/).

Reply:
Moreover, use `\vect{}` for all vectors in the text, here and elsewhere.